System/Posix/Env/ByteString.hsc

```diff
@@ -133,7 +133,7 @@ putEnv (PS fp o l) = withForeignPtr fp $ \p -> do
   --
   -- hence we must not free the buffer
   buf <- mallocBytes (l+1)
-  memcpy buf (p `plusPtr` o) l
+  copyBytes buf (p `plusPtr` o) l
```

Comment:
I can confirm that `copyBytes` exists since at least [base-4.9.1.0](https://hackage.haskell.org/package/base-4.9.1.0/docs/Foreign-Marshal-Utils.html#v:copyBytes).